python/4_lens.i

```diff
@@ -83,36 +79,30 @@
 #ifdef WITHNUMPY
 %extend crpropa::ParticleMapsContainer {
   PyObject *addParticles(PyObject *particleIds,
-                PyObject *energies,
-                PyObject *galacticLongitudes,
-                PyObject *galacticLatitudes,
-                PyObject *weights)
-  {
+                        PyObject *energies,
+                        PyObject *galacticLongitudes,
+                        PyObject *galacticLatitudes,
+                        PyObject *weights) {
     //ToDo: Handle strided arrays
 
     //ToDo: Check that input objects are arrays  PyArray_Check
```

Comment:
Is this still an open ToDo? If yes we should open an issue for that. 
In any case we should not write ToDo statements in the comments.

Reply:
I actually don't know whose to-do list this is. (Not mine).
I will just remove it then.